primes.rb

```diff
@@ -1,6 +1,8 @@
+require "prime"
 class Primes
   def self.sum_to(limit = 100)
-    # TODO - add your prime number solution here...
-    "I'm working on it!"
+	  list_of_primes = []
+	  Prime.each(limit) { |prime| list_of_primes << prime }
+	  return list_of_primes.reduce(:+)
```

Comment:
This works but also loses the opportunity to play around with some good prime number algorithms